### PR TITLE
Add first draft of BBS vignette

### PR DESCRIPTION
**Pull request by ethanwhite**

### CODE REVIEW

**Comment by ethanwhite** [comment]
@henrykironde - here's a slightly cleaned up version of the material I use for class. Feel free to merge and then keep working on it from here. I flagged a spot to potentially include the provenance functionality.

File: paper/vignettes/breed-bird-survey-analysis.Rmd [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+```{r}
+ggplot(richness_w_env, aes(x = bio12, y = richness)) +
+  geom_point() +
+  geom_smooth() +
+  facet_wrap(~statenum, scales = 'free')
+```
+
+## Provenance
+
+HENRY MAYBE ADD A SHORT EXAMPLE HERE OF USING THE PROVENANCE FEATURES TO ENSURE THAT WE CAN REPLICATE THE ANALYSIS LATER?
```
Comment (henrykironde) [inline paper/vignettes/breed-bird-survey-analysis.Rmd:123]: OK

**Comment by ethanwhite** [comment]
OK @henrykironde - I think this is a fully functioning vignette at this point. I pulled the provenance stuff due to https://github.com/weecology/retriever/issues/1524 since we don't want to have to wait on a new release of the core package for this.

**Comment by henrykironde** [comment]
Thanks @ethanwhite, Am looking into that error on `utf-8`

**Pull request merged**